docs/specification/draft/basic/authorization.mdx

```diff
@@ -91,6 +91,33 @@ as described in [RFC9728 Section 5.1 "WWW-Authenticate Response"](https://datatr
 
 MCP clients **MUST** be able to parse `WWW-Authenticate` headers and respond appropriately to `HTTP 401 Unauthorized` responses from the MCP server.
 
+#### Default Scopes
+
+In addition to the standard Protected Resource Metadata fields defined in [RFC9728](https://datatracker.ietf.org/doc/html/rfc9728),
+MCP servers **MAY** include the optional `scopes_default` extension.
+
+`scopes_default` is a JSON array containing a list of scope values that represent the default scopes for clients to use
+when initiating the authorization flow. This field provides guidance to clients about the appropriate scopes to request
+when starting the authorization process or responding to scope-related errors.
+
+MCP servers **SHOULD** include this field to follow the principle of least privilege and prevent clients from requesting excessive permissions.
+By providing `scopes_default`, servers guide clients toward requesting only the minimum scopes necessary for typical operations,
+rather than all available scopes.
+
+The `scopes_default` values **MUST** be a subset of the `scopes_supported` values.
```

Comment:
In a typical implementation, how to determine the `scopes_default`?

Once the client discovers the available tools, tool selection and execution will be dynamic based on the reasoning at the host side.

Reply:
Resolving since the suggested approach changed. Recommend checking #850 for exploration on primitive authorization.